service/rest_build.go

```diff
@@ -51,6 +52,13 @@ func (restapi *restAPI) getBuildInfo(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	tasks, err := task.FindAll(task.ByBuildId(b.Id).WithFields(task.StatusKey, task.TimeTakenKey, task.DisplayNameKey))
```

Comment:
same comment here about ByBuildId versus TaskIds

Reply:
we have an index on build_id